src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/AutoSuggestBoxTests/AutoSuggestBox_SoftKeboard.xaml.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Linq;
+using Uno.UI.Samples.Controls;
+using Windows.UI.Xaml.Controls;
+
+// The User Control item template is documented at https://go.microsoft.com/fwlink/?LinkId=234236
+
+namespace UITests.Windows_UI_Xaml_Controls.AutoSuggestBoxTests;
+
+[SampleControlInfo("AutoSuggestBox",
+				   "AutoSuggestBox_SoftKeboard",
+					description: "This test needs the Soft Keyboard",
```

Comment:
Can you expand on the test expectations and steps?

Reply:
updated